### PR TITLE
TAJO-1396 Unexpected IllegalMonitorStateException can be thrown in QueryInProgress

### PR DESCRIPTION
**Pull request by navis**
Simple / trivial patch

### CODE REVIEW

File: tajo-core/src/main/java/org/apache/tajo/master/QueryInProgress.java [hunk 1]
```diff
@@ -163,6 +168,12 @@ public void submmitQueryToMaster() {
 
     try {
       writeLock.lockInterruptibly();
+    } catch (Exception e) {
```
Comment (jihoonson) [inline tajo-core/src/main/java/org/apache/tajo/master/QueryInProgress.java]: I have a question. 
Would you explain why writeLock.lockInterruptibly() is called subsequently?

Comment (navis) [inline tajo-core/src/main/java/org/apache/tajo/master/QueryInProgress.java]: It just excluded locking from try/finally clause, avoiding releasing lock which is not held by the thread.

File: tajo-core/src/main/java/org/apache/tajo/master/QueryInProgress.java [hunk 2]
```diff
@@ -163,6 +168,12 @@ public void submmitQueryToMaster() {
...
+      LOG.error(e.getMessage(), e);
+      return;
+    }
+    try {
+      writeLock.lockInterruptibly();
```
Comment (navis) [inline tajo-core/src/main/java/org/apache/tajo/master/QueryInProgress.java]: Ah, you are saying on this. Forgot to remove it, sorry

**Comment by navis** [comment]
Added more log messages

**Comment by jinossy** [comment]
+1 This PR will remove the possible IllegalMonitorStateException

**Pull request closed without merging**